include/smooth/feedback/asif_func.hpp

```diff
@@ -190,7 +195,7 @@ void asif_to_qp_update(
     // insert barrier constraint
     const Eigen::Matrix<double, nh, nx> dh_dx0 = dh_dx * dx_dx0;
     qp.A.template block<nh, nu>(k * nh, 0)     = dh_dx0 * d_f0_du;
-    qp.l.template segment<nh>(k * nh)          = -dh_dt - prm.alpha * hval - dh_dx0 * f0;
```

Comment:
The variable in the QP is mu := u - u_des.

I think the change would be correct if the variable is u, but for mu I think it's good the way it is.